README.md

```diff
@@ -84,8 +97,9 @@ tomcat::instance { 'default':
 ```puppet
 class { 'java': }
 
-tomcat::install { '/opt/tomcat9':
-  source_url => 'https://www.apache.org/dist/tomcat/tomcat-9/v9.0.x/bin/apache-tomcat-9.0.x.tar.gz'
+tomcat_install { '/opt/tomcat9':
```

Comment:
I would really prefer for the `tomcat::install` defined type to remain the public interface. It should wrap the `tomcat_install` native time to receive "super powers" but still offer feature compatibility otherwise.

This will give a universal interface for use of the module still without adding parallel incompatible usages. It would be great to see all of the capabilities of `tomcat::install` implemented in the native type, but that's probably asking a bit much.

Reply:
> I would really prefer for the `tomcat::install` defined type to remain the public interface. It should wrap the `tomcat_install` native time to receive "super powers" but still offer feature compatibility otherwise.

The `tomcat::install` defined type is still using the archive resource; this can be swapped out for `tomcat_install` yeah? That would be great!